### PR TITLE
feat: add Avatar HD preset

### PR DESCRIPTION
**Pull request by jappyjan**
I added a preset to configure MSP OSD for Walksnail Avatar HD VTX's

### CODE REVIEW

**Comment by limonspb** [comment]
> I added a preset to configure MSP OSD for Walksnail Avatar HD VTX's

Hi
Have you read the tips for the authors in the main readme? Thanks

**Review by limonspb — CHANGES_REQUESTED** [review]
Hi

Could you please check the tips for authors:
https://github.com/betaflight/firmware-presets#tips-for-authors

we are asking not to include index updates in the PR
Thanks!

File: presets/4.3/vtx/Avatar_HD.txt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+set displayport_msp_serial = 2
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): use displayport on uart 4
+serial 3 1 115200 57600 0 115200
+set displayport_msp_serial = 3
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): use displayport on uart 5
+serial 4 1 115200 57600 0 115200
```
Comment (limonspb) [inline presets/4.3/vtx/Avatar_HD.txt:38]: what this command is doing exactly? Does it work in 4.3 and 4.2?

Comment (jappyjan) [reply presets/4.3/vtx/Avatar_HD.txt:38]: here is the definition of that command.
https://github.com/betaflight/betaflight/blob/master/docs/Serial.md

and yes, as far as i know it works just fine on both BF versions

**Comment by jappyjan** [comment]
sorry for including the index updates, guess i have missed the note on not to include them. just fixed that.

**Comment by limonspb** [comment]
> sorry for including the index updates, guess i have missed the note on not to include them. just fixed that.

Hi, now you have two commits. It is also mentioned in the tips :)

**Comment by sugaarK** [comment]
why did you close @jappyjan

**Comment by jappyjan** [comment]
i did not. i only pushed my changes, came back here and it was closed

**Comment by jappyjan** [comment]
just saw it says i did it... dont know why?!

**Comment by sugaarK** [comment]
only thing I would ask is to add the 4.4 tag as well as this will be the same

**Comment by limonspb** [comment]
> just saw it says i did it... dont know why?!

I guess because "GitHub desktop" :) I never used it, so dunno

**Comment by limonspb** [comment]
yeah looks good now, if you don't mind adding 4.4 firmware? But idk how to do it smooth in github desktop, i only know git bash

**Comment by jappyjan** [comment]
ah no, i use git cli for the standard git stuff and just the github website for managing PR's and stuff ^^

**Comment by jappyjan** [comment]
do i need to change any of the commands for 4.4?
did not yet look into changelog/documentation of 4.4...

**Comment by sugaarK** [comment]
> do i need to change any of the commands for 4.4?
> did not yet look into changelog/documentation of 4.4...

nah none of this has changed at this point

**Comment by limonspb** [comment]
> do i need to change any of the commands for 4.4? did not yet look into changelog/documentation of 4.4...

based on our MSP-OSD expert @sugaarK there will be no changes for 4.4 here

**Comment by jappyjan** [comment]
is this enough or do i need to copy/paste this to the 4.4 folder as well?

**Comment by sugaarK** [comment]
leave it in the 4.3 folder. as its not just a 4.4 preset

**Comment by jappyjan** [comment]
should be good to go then

**Review by sugaarK — APPROVED** [review]

**Pull request merged**